pkg/acceptance/allocator_test.go

```diff
@@ -278,16 +290,14 @@ func (at *allocatorTest) ValidateSchemaChanges(
 		_ = db.Close()
 	}()
 
-	var now string
-	if err := db.QueryRow("SELECT cluster_logical_timestamp()").Scan(&now); err != nil {
-		t.Fatal(err)
-	}
+	now := timeutil.Now()
+	nowString := fmt.Sprintf("%d.0000000000", now.UnixNano())
```

Comment:
I had to stop using the cluster logical time because a cluster logical time is a decimal which is read as a string, which then needs to be converted to a decimal and to a time.Time for this change. Since we already have a 5 second sleep above i thought this was fine.